charts/osdfir-infrastructure/README.md

```diff
@@ -240,6 +241,27 @@ kubectl delete pvc -l release=osdfir-release
 | `turbinia.redis.replica.resources.limits`   | The resources limits for the Redis replica containers                                        | `{}`             |
 | `turbinia.redis.replica.resources.requests` | The requested resources for the Redis replica containers                                     | `{}`             |
 
+### Oauth2 Proxy configuration parameters
```

Comment:
Add a link to the official oauth2 documentation with descriptions of the oauth2 settings?